src/widgets/DataTable/DataTableRow.js

```diff
@@ -83,10 +84,13 @@ const DataTableRow = React.memo(
           // Indicator if the right hand border should be removed from styles for this cell.
           const isLastCell = index === columns.length - 1;
 
-          // This cell is disabled if the pageObject is finalised, the row has been explicitly set
-          // as disabled, or the rowData is disabled (i.e. data is an invoice),
-          const isDisabled =
-            isFinalised || (rowState && rowState.isDisabled) || rowData.isFinalised;
+          // This cell is disabled if:
+          // - the pageObject is finalised
+          // - the row has been explicitly set as disabled
+          // - the rowData is disabled (i.e. data is a finalised invoice)
+          const { isFinalised: isFinalisedPage = false } = rowData ?? {};
```

Comment:
If I'm reading this correctly, the global `isFinalised` relates to the page and this one relates to the row

```suggestion
          const { isFinalised: isFinalisedRow= false } = rowData ?? {};
```

Reply:
0ddf593d0ee261753ca1e7246acb131e7d749000.